src/main/electron/ClusterContent.js

```diff
@@ -0,0 +1,102 @@
+var app = angular.module('ClusterApp', []);
+
+app.config(['$locationProvider', function($locationProvider) {
+	$locationProvider.html5Mode(true);
+}]);
+app.controller('ClusterController', ['$scope', '$location', function($scope, $location) {
+
+	console.log($location.url());
+	console.log($location.search().data);
+	console.log($location.search().cluster);
+
+	var fs = require('fs');
+	var content;
+	var data = fs.readFileSync($location.search().data, 'utf8');
+	content = parseCSVFile(data);
+	for (var i in content.features) {
+		if (content.features[i].properties.id == $location.search().cluster) {
+			$scope.ID = content.features[i].properties.id;
+			$scope.size = parseFloat(content.features[i].properties.size);
+			$scope.infected = parseFloat(content.features[i].properties.infected);
+			$scope.infected_percent = parseFloat(content.features[i].properties.infected_percent);
+			$scope.cluster_type = content.features[i].properties.type;
+			$scope.lat = parseFloat(content.features[i].geometry.coordinates[1]);
+			$scope.lon = parseFloat(content.features[i].geometry.coordinates[0]);
+			if ($scope.infected_percent == -1) {
+				$scope.infected_percent = 0;
+			}
+		}
+	}
+	$scope.title = "Cluster " + $scope.ID;
+	$scope.currentDay = $location.search().currentDay;
+
+	var files = [];
+	var filenames = [];
+	var clusterEvolution = [];
+	var dircontent = fs.readdirSync($location.search().directory);
+	dircontent.forEach( function (file){
+		var data = fs.readFileSync($location.search().directory + "/" + file, 'utf8');
+		files.push(data);
+		filenames.push(file);
+	});
+
+	clusterEvolution = getClusterInfectedCourse(files, $scope.ID);
+
```

Comment:
ik zie van een groot deel van de "copypasted" dingen de duplicate-heid niet in ?